src/plugins/data/common/search/aggs/metrics/percentile_ranks.ts

```diff
@@ -95,5 +95,11 @@ export const getPercentileRanksMetricAgg = ({
     getValue(agg, bucket) {
       return getPercentileValue(agg, bucket) / 100;
     },
+    getValueBucketPath(aggConfig) {
+      if (aggConfig.key) {
+        return `${aggConfig.id}`;
+      }
+      return `${aggConfig.id}.${aggConfig.params.values[0]}`;
+    },
```

Comment:
ℹ️ In case of `filtered_metric` the aggregation returned has a different format (IaggConfig and not IResponseAggConfig)